<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoTargetMark</class>
 <widget class="QWidget" name="VideoTargetMark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>320</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(203, 203, 203);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="DisplayWidget" name="widget_display" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_open_video">
        <property name="text">
         <string>打开视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save_target">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>保存目标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_video_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cur_frame">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_target_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_first_frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>首帧</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_last_frame_auto">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>上一帧的标记为空时
使用当前帧的标记</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_last_frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>上一帧</string>
        </property>
        <property name="shortcut">
         <string>R</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_next_frame_auto">
        <property name="text">
         <string>下一帧的标记为空时
使用当前帧的标记</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_next_frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>下一帧</string>
        </property>
        <property name="shortcut">
         <string>F</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>40</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header>displaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
